### PR TITLE
Publicize: Notify about broken connections

### PR DESCRIPTION
**Pull request by artpi**
Presents notices about broken connections in republicize window

screen shot 2016-12-01 at 18 59 26 pm

## Testing instructions

1. Set up publicize, set up accounts (ex: twitter)
2. Log in to twitter and break the connection [Revoke access] here: https://twitter.com/settings/applications
3. Go to /sharing/ to see if indeed there is trouble connecting
4. Go to publicize and see notice about twitter being disconnected
5. Would be nice to check for other services :)

CC @enejb @gziolo @bikedorkjon @gwwar @drw158

### CODE REVIEW

**Comment by enejb** [comment]
I tried testing this and I ran into the following issue. 
See the screenshot.
screen shot 2016-12-01 at 11 36 57

The account that has problem connecting is same one that successfully send out the message. 
Can we maybe check in calypso if we tried to publicize to a broken account? Before showing a success message. 

Ideally this would be fixed on the api side. (It should return you true results) Right now it just tells you that it successfully send the requests to the services.

**Comment by gziolo** [comment]
Another option would be to disable broken connections and mark them with yellow color :)

**Comment by artpi** [comment]
I made broken connection show and act as inactive:
screen shot 2016-12-20 at 17 19 27 pm

File: client/my-sites/posts/post-share.jsx [hunk 1]
```diff
@@ -157,7 +156,22 @@ const PostSharing = React.createClass( {
...
+					{ this.hasConnections() && <div>
+						<div>
+							{ this.props.connections
+								.filter( connection => connection.status === 'broken' )
+								.map( connection => <Notice
+									key={ connection.keyring_connection_ID }
+									status="is-warning"
+									showDismiss={ false }
+									text={ this.translate( 'There is an issue connecting to %s.', { args: connection.label } ) }
+								>
```
Comment (gziolo) [inline client/my-sites/posts/post-share.jsx]: We might consider giving more context for the translators.

**Comment by gziolo** [comment]
It would be nice to add margin between notices when we have more than one information about broken connections:
screen shot 2016-12-21 at 16 57 39

Another option is to group all service names into one message.

We should also change cursor icon when user hovers disabled connection option. Another idea would be to simply remove it from list at all. User sees this big yellow message anyway so we don't need to duplicate this information.

**Comment by gziolo** [comment]
As discussed on Slack. Let's add margin between yellow notices, and different cursor indicator for disabled services. Other than that it is ready to go 🚢 

Great work on Publicize @artpi!

**Comment by gziolo** [comment]
Still good after changes made 👍

**Pull request merged**